NoSoft-SA/nspack
# added check for builtup pallets on buildup sheets

Summary of this change:
Block pallets that have the `built_up` flag set to true from being added to a govt inspection sheet that does not have the `buildup_inspection` flag also set to true.

Edit: added TITAN bugfix causing `ecert_agreement_code` to not be populated correctly.

CHANGELOG entry:

### Added
- `buildup_sheet_for_buildup_pallet_check` in pallet task permission check.

How to test this code via the UI:
Try to add a buildup pallet to a normal govt inspection sheet and confirm it fails.
Adding it to a buildup sheet should pass.
